libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -334,6 +331,17 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
     }
   }
 
+  public get changedProjectsOnly(): boolean | undefined {
+    return this._changedProjectsOnlyParameter?.value;
+  }
+
+  public set changedProjectsOnly(value: boolean) {
+    if (this._changedProjectsOnlyParameter) {
+      // Mutate the parameter so that the telemetry reflects the value.
+      (this._changedProjectsOnlyParameter as unknown as { _value?: boolean })._value = value;
```

Comment:
This is a little iffy... let's replace this with a local variable `_changedProjectsOnly` that takes precedence over the parameter value in the getter:
```typescript
  public get changedProjectsOnly(): boolean | undefined {
    return this._changedProjectsOnly ?? this._changedProjectsOnlyParameter?.value;
  }
```